app/backup.go

```diff
@@ -11,12 +11,23 @@ import (
 
 	"github.com/rancher/longhorn-engine/sync"
 	"github.com/rancher/longhorn-engine/util"
+	"strings"
 )
 
 func BackupCmd() cli.Command {
 	return cli.Command{
 		Name:      "backups",
 		ShortName: "backup",
+		Flags: []cli.Flag{
```

Comment:
I don't think we need those new parameters. We should able to pick them up in the environment variable instead.